cmd/ebpf-instrument/main.go

```diff
@@ -99,6 +93,41 @@ func main() {
 	}
 }
 
+func logConfig(config *obi.Config) {
```

Comment:
nit: can we early exit if config.LogOption is empty string and only marshal to YAML once to avoid code repetition?